kolko.py

```diff
@@ -0,0 +1,66 @@
+
+def printer(tab):
+    'print table with positions to choose'    
+    print('{}\n{}\n{}'.format(tab[0],tab[1],tab[2]))
+
+def instert_to_tab(player,n):
+    '''inserting player to table return update table
+        player - X or O
+        n - position number'''    
+    c=0
+    for arr in tab:
+        for i,pos in enumerate(arr):
+            if n==c:
+                arr[i]=player
+            c += 1
+    return tab
+
+def winner(tab):    
+    '''check who win. Return True when anybody win.
+    tab - used_nums'''
+    for arr in way_to_win:
+        if players[0]==tab[arr[0]]==tab[arr[1]]==tab[arr[2]] or players[1]==tab[arr[0]]==tab[arr[1]]==tab[arr[2]]:
+            return True
+
+def draw(tab):
+    '''if draw True
+    tab - used_nums'''
+    return all([ele for ele in tab])
+
+# table to view
+tab = [['1','2','3'],
+       ['4','5','6'],
+       ['7','8','9']]
+# list with choosen positions
+used_nums = ['','','','','','','','','']
+# players
+players = ['X','O']
```

Comment:
A może by tak zmienić żeby zmienna player przyjmowała wartości 'X' i '0' zamiast 0 i 1 a potem bawić się z zamianą na znaki?

Reply:
no można to zastosować.